lib/src/builder_impl.dart

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:analyzer/dart/element/element.dart';
+import 'package:build/build.dart';
+import 'package:dart_style/dart_style.dart';
+
+import 'generated_output.dart';
+import 'generator.dart';
+import 'library.dart';
+import 'utils.dart';
+
+const defaultFileHeader = '''// GENERATED CODE - DO NOT MODIFY BY HAND
+
+''';
+
+/// A [Builder] wrapping on one or more [Generator]s.
+class BuilderImpl extends Builder {
```

Comment:
why does this need to be in a different file?

why does this need to be named like "Impl"?

Reply:
+1. This change isn't valuable.